src/stac_utils/google.py

```diff
@@ -1,14 +1,18 @@
+import io
 import json
 import logging
 import os
 import sys
 
+import pandas
```

Comment:
i actually really want to keep pandas out of this file so that we can keep it as an optional dependency (it's kind of huge and sometimes risks preventing AWS Lambda deployment). would you mind moving get_dataframe_from_drive to the pandas_utils.py file?